.aws/faucet.sh

```diff
@@ -17,7 +19,8 @@ do
 
             # Just in case we were running this command with rest-server switched on, get again the address
             ADDRESS=$(./gaiacli keys show ${ACCOUNT} --home . --address)
-            echo ${PASSWORD} | ./gaiacli tx send ${DESTINATION} ${AMOUNT} --home . --from ${ADDRESS} --chain-id=${NETWORK}
+            echo ${PASSWORD} | ./gaiacli tx send ${DESTINATION} ${AMOUNTS} --home . --from ${ADDRESS} --chain-id=${NETWORK}
+#            echo ${PASSWORD} | ./gaiacli tx send ${DESTINATION} ${AMOUNTP} --home . --from ${ADDRESS} --chain-id=${NETWORK}
```

Comment:
ditto

Reply:
datto

Reply:
I re-enalbed those, apparently the drainig of money always happened at `stakes`